packages/api/orm/index.ts

```diff
@@ -0,0 +1,16 @@
+import { createConnection } from "typeorm";
+import { Portfolio } from "./entity/portfolio";
+import { ProvisioningStatus } from "../models/ProvisioningStatus";
+import "reflect-metadata";
+
+createConnection()
+  .then(async (connection) => {
+    const pd = new Portfolio();
+    pd.status = ProvisioningStatus.IN_PROGRESS;
+    await connection.manager.save(pd);
+    console.log("Saved a new portfolio with id: " + pd.id);
+
+    const pfs = await connection.manager.find(Portfolio);
+    console.log("Loaded portfolios: ", pfs);
+  })
+  .catch((error) => console.log(error));
```

Comment:
As I've been working in my #434 PR I've been using `await` rather than `.then()`/`.catch()` to match some of our existing style across the app. Do you have a preference going forward?

Reply:
Await is fine.  The TypeORM guides use this `then()` / `catch()` syntax.